.github/workflows/post-commit.yml

```
@@ -66,6 +68,7 @@ jobs:
         if: always()
   bats:
     runs-on: ubuntu-18.04
+    if: github.event_name != 'pull_request' || github.event.pull_request.draft == false
```

Is it possible to create an environment variable and then use it everywhere?

```suggestion
    env:
         skipCI = ${{ github.event_name != 'pull_request' || github.event.pull_request.draft == false }}
    if: ${{skipCI}}
```

---

Thanks @dineshchitlangia for the review.

Environment variables cannot be used in `jobs.<name>.if`, at least I ran into error trying to do so: `Unrecognized named-value: 'env'`.

Jobs can use their dependencies' output.  We can create a job that produces output and make all other jobs depend on it.  I am planning to do this for implementing selective CI checks (HDDS-5000).  We can make a step in that direction as part of this PR, but I'm afraid it might seem overkill for reviewers. ;)  Here's what it would look like: https://github.com/adoroszlai/hadoop-ozone/commit/802c6f963402f76ce6ca20055fa4a8f23aeee6f3

---

Thanks @adoroszlai , that makes sense. All clear from my end :)

---

> All clear from my end :)

Thanks.  Do you mean the current version in this PR is OK, or the one in the other branch (https://github.com/adoroszlai/hadoop-ozone/commit/802c6f963402f76ce6ca20055fa4a8f23aeee6f3)?

---

Current version.